utilities/vcf2json.py

```diff
@@ -226,6 +233,8 @@ def vcf2json(vcf_filename=None, ref_build=None, patient_id=None,
         output_json["genomicSourceClass"] = genomic_source_class
         onRef = 1
 
+        output_json = extractINFOField(output_json, record, codeDict)
```

Comment:
Dictionaries are passed by reference. You can omit the return from `extractINFOField` and `parseANN`. You just need to call these with `output_json`.

Reply:
```suggestion
        extractINFOField(output_json, record, codeDict)
```